.github/workflows/ci.ubuntu-x64.dev.yml

```
@@ -0,0 +1,54 @@
+    runs-on: ubuntu-latest
+    name: Docker CI (ubuntu-x64.dev)
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v4
+
+    - name: Cache Docker layers
+      uses: actions/cache@v4
+      with:
+        path: /tmp/.docker-cache
+        key: docker-${{ runner.os }}-${{ runner.os-version }}-${{ hashFiles('.devcontainer/Docker.ubuntu-x64.dev') }}
```

actionlint (1.7.7)

property "os-version" is not defined in object type {arch: string; debug: string; environment: string; name: string; os: string; temp: string; tool_cache: string}

(expression)